ksql-engine/src/test/java/io/confluent/ksql/schema/ksql/inference/DefaultSchemaInjectorFunctionalTest.java

```diff
@@ -506,7 +506,7 @@ private void shouldInferSchema(
 
     // Then:
     final Statement withSchema = KsqlParserTestUtil
-        .buildSingleAst(inferred.getStatementText(), metaStore)
+        .buildSingleAst(inferred.getMaskedStatementText(), metaStore)
```

Comment:
This seems to parse the sql statement, should it be unMasked?

Reply:
Good catch, thanks! The reason this did not pop up in testing is its only testing "CREATE STREAM" currently.